.eslintrc

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "loopback"
-}
+    "extends": "loopback",
+    "rules": {
+      "max-len": ["error", 120, 4, {
+        "ignoreComments": true,
+        "ignoreUrls": true,
+        "ignorePattern": "^\\s*var\\s.=\\s*(require\\s*\\()|(/)"
+      }],
+      "camelcase": 0,
+      "one-var": "off"
```

Comment:
@loay i did the change instead, so you dont have to rebase against my globalize pr